kilkari/src/main/resources/META-INF/motech/applicationContext.xml

```diff
@@ -31,4 +40,17 @@
             </list>
         </property>
     </bean>
+
+    <cache:annotation-driven />
+
+    <bean id="cacheManager" class="org.springframework.cache.support.SimpleCacheManager">
```

Comment:
I guess the good thing is doesn't really matter just yet as long as we get the annotations right.
#notansweringthequestion